repondeur/zam_repondeur/models/amendement.py

```diff
@@ -257,10 +257,10 @@ def article_url_key(self) -> str:
             f"{self.article.mult}.{self.article.pos}"
         )
 
-    def grouping_key(self) -> str:
+    def grouping_key(self) -> Union[str, Tuple[str, str, str]]:
```

Comment:
Maybe we can return `("", "", "")` to keep the type simple?

Reply:
You mean `(self.num_str, "", "")` for governmental ones?

Reply:
Yes, sorry